andela/ah-titans-frontend
# #160226378 Implement login design

#### What does this PR do?
Design Login UI with the use of materialize.

#### Description of tasks to be completed
Implement designs for the Signup/Registration Page.

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ git checkout ft-implement-login-design-160226378
```
```sh
$ cd ah-titans-frontend
```

>Install the dependencies using the command below.

```sh
$ npm install
```
>Run the application as below.
```sh
$ npm start
```
>Now you can navigate to your browser and you should be able to see a page as the one shown on the screenshot below.

<img width="1280" alt="screen shot 2018-09-03 at 17 24 16" src="https://user-images.githubusercontent.com/12511966/44991884-427d4680-af9e-11e8-9de3-3a4c59dc0be0.png">


#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160226378](https://www.pivotaltracker.com/story/show/160226378)
